lms/migrations/versions/7a4812876915_fix_user_id_unique_constraint.py

```diff
@@ -0,0 +1,40 @@
+"""
+Fix user_id unique constraint.
+
+Revision ID: 7a4812876915
+Revises: 6882c201b56e
+Create Date: 2022-01-25 16:07:34.805966
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = "7a4812876915"
+down_revision = "6882c201b56e"
+
+
+def upgrade():
+    op.drop_constraint("uq__user__application_instance_id", "user")
```

Comment:
Presumably you had to look up this name anyway? Which makes some of the claims of convenience a little less convincing...